src/OrbitGl/Track.cpp

```diff
@@ -232,6 +232,6 @@ void Track::OnCollapseToggle(bool /*is_collapsed*/) { RequestUpdate(); }
 void Track::OnDrag(int x, int y) {
   CaptureViewElement::OnDrag(x, y);
 
-  pos_[1] = mouse_pos_cur_[1] - picking_offset_[1];
+  SetPos(GetPos()[0], mouse_pos_cur_[1] - picking_offset_[1]);
```

Comment:
This is the main change that fixes the bug, right? The rest is "only" because `pos_` is now a private variable?

I think this makes a lot of sense, just want to be sure I understand this correctly.

Reply:
Yes, exactly. I splitted it into two commits, and the first one is only that line.